<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Analytics</class>
 <widget class="QDialog" name="Analytics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1123</width>
    <height>853</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TATSSI QA Analytics</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>820</y>
     <width>166</width>
     <height>24</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="grpboxInputParam">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1101</width>
     <height>401</height>
    </rect>
   </property>
   <property name="title">
    <string>Input parameters</string>
   </property>
   <widget class="QPushButton" name="pbOutputDirectory">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>1081</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select input data directory</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtProduct">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;MOD13A2&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblProduct">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Product</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTile">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Version</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtVersion">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;006&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tvQADef">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>1081</width>
      <height>192</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectItems</enum>
    </property>
    <column>
     <property name="text">
      <string>ProductAndVersion</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>QualityLayer</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Value</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Description</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Acceptable</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Length</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QComboBox" name="cmbQADef">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>321</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblQADef">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>150</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>QA definition</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblStartDate">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>80</y>
      <width>142</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Start date (dd-mm-yyy)</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="start_date">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>80</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string notr="true">dd-MM-yyyy</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblEnbDate">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>80</y>
      <width>135</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>End date (dd/mm/yyy)</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="end_date">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>80</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string notr="true">dd-MM-yyyy</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDataDir_">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>58</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>true</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Data directory:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbGetQADefinitions">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>1081</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Get time series QA definitions</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDataDir">
    <property name="geometry">
     <rect>
      <x>116</x>
      <y>56</y>
      <width>971</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>true</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>/home/glopez/Projects/TATSSI/data/MOD13A2.006</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpboxAnalytics">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>1101</width>
     <height>381</height>
    </rect>
   </property>
   <property name="title">
    <string>Analytics</string>
   </property>
   <widget class="QLabel" name="lblQAParamName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>QA parameter name</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbQAParamName">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblQADescription">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSelectAll">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>320</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select ALL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbPlotMaxGapLength">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>320</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot Max Gap Length</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbQAAnalytics">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>320</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>QA Analytics</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSaveQAAnalytics">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save QA Analytics</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbLoadQAAnalytics">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>320</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load QA Analytics</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>1081</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>1081</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QListWidget" name="lwQAParamDesc">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>681</width>
      <height>271</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tbrSelection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>391</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblSelection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Selection</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Analytics</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Analytics</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
